src/js/plugins/plugin.resume.js

```diff
@@ -0,0 +1,88 @@
+/* global BookReader */
+
+/**
+ * Plugin to remember the current page number in a cookie
+ */
+jQuery.extend(BookReader.defaultOptions, {
+  enablePageResume: true,
+  /**
+   * @const null|string eg '/', '/details/id'
+   */
+  resumeCookiePath: null,
+});
+
+/**
+ * @override BookReader.init
+ */
+BookReader.prototype.init = (function(super_) {
```

Comment:
We don't need to duplicate the name of the function here; also docstring can be single line, like so:

```suggestion
/** @override */
BookReader.prototype.init = (function(super_) {
```